src/Autofac/ILifetimeScope.cs

```diff
@@ -70,9 +82,30 @@ public interface ILifetimeScope : IComponentContext, IDisposable, IAsyncDisposab
         /// will be disposed along with it.
         /// </summary>
         /// <remarks>
-        /// The components registered in the sub-scope will be treated as though they were
-        /// registered in the root scope, i.e., SingleInstance() components will live as long
-        /// as the root scope.
+        /// <para>
+        /// Any components registered in the sub-scope will only be resolvable from this new scope,
+        /// or additional child scopes subsequently resolved from it.
+        /// </para>
+        /// <para>
+        /// Components registered in the sub-scope that provide services already registered
+        /// in a parent <see cref="ILifetimeScope"/> (or root <see cref="IContainer"/>) will
+        /// override that same service registration when resolving instances.
+        /// </para>
+        /// <para>
+        /// If you resolve an <see cref="IEnumerable{TService}"/> in the new scope, the returned set of components will
```

Comment:
```suggestion
        /// If you resolve an <see cref="System.Collections,Generic.IEnumerable{TService}"/> in the new scope, the returned set of components will
```
Compiler is complaining about not finishing IEnumerable 😩

Reply:
Resolved this problem manually.